contracts/handlers/HandlerHelpers.sol

```diff
@@ -49,6 +49,14 @@ contract HandlerHelpers is IERCHandler {
         return resourceID;
     }
 
+    /**
+        @notice Used to manually release funds from ERC safes.
+        @param tokenAddress Address of token contract to release.
+        @param recipient Address to release tokens to.
+        @param amountOrTokenID Either the amount of ERC20 tokens or the ERC721 token ID to release.
+     */
+    function withdraw(address tokenAddress, address recipient, uint256 amountOrTokenID) public virtual override {}
```

Comment:
Not sure if this is the best way to solve this issue, but `HandlerHelpers.sol` inherits from `IERCHandler.sol` which now contains the `withdraw` method (so it can be called via `Bridge.sol`), which means `HandlerHelpers.sol` needs to implement the method, but can't because each ERC handler has a different implementation for the `withdraw` method (e.g. `releaseERC20` v.s. `releaseERC721`). So by adding the `virtual` keyword, it enforces whatever contract inherits `HandlerHelpers.sol` must override the `withdraw` method